### PR TITLE
[brainfuck/cs] Brainfuck cs_CZ translation

### PR DESCRIPTION
**Pull request by vojtasvoboda**
Brainfuck cs_CZ translation

### CODE REVIEW

File: cs-cz/brainfuck.html.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+language: brainfuck
+contributors:
+    - ["Prajit Ramachandran", "http://prajitr.github.io/"]
+    - ["Mathias Bynens", "http://mathiasbynens.be/"]
+translators:
+    - ["Vojta Svoboda", "https://github.com/vojtasvoboda/"]
+filename: learnbrainfuck.bf
```
Comment (vendethiel) [inline cs-cz/brainfuck.html.markdown]: please add a `-cz` suffix to the filename

**Comment by vojtasvoboda** [comment]
Suffix added

**Comment by vendethiel** [comment]
Thanks! Big sorry for the delay here...

**Comment by vendethiel** [comment]
:star:

**Pull request merged**